src/components/NodeMovingPlaceholder/index.test.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import NodeMovingPlaceholder from './index';
+
+const svg = document.createElement('svg');
+
+const defaultProps = {
+  node: {
+    id: 'node-id',
+    states: [],
+    parents: [],
+    position: { x: 0, y: 0 },
+    size: { width: 160, height: 100 },
+  },
+  svg,
+  onSetPosition: jest.fn(),
+  onCancel: jest.fn(),
```

Comment:
Avoid creating jest stubs like this in a shared state object, it will make tests influence each other, which can take to intermittent tests. You could do something like this instead:

```js
describe('NodeMovingPlaceholder Component', () => {
  let defaultProps;

  beforeEach(() => {
    defaultProps = {
      node: {
        id: 'node-id',
        states: [],
        parents: [],
        position: { x: 0, y: 0 },
        size: { width: 160, height: 100 },
      },
      svg,
      onSetPosition: jest.fn(),
      onCancel: jest.fn(),
    };
  });
// ...
});
```

This way it'll be a new object for every test, making it safer.